ip-api/operations.py

```diff
@@ -0,0 +1,77 @@
+""" Copyright start
+  Copyright (C) 2008 - 2022 Fortinet Inc.
+  All rights reserved.
+  FORTINET CONFIDENTIAL & FORTINET PROPRIETARY SOURCE CODE
+  Copyright end """
+
+from connectors.core.connector import get_logger, ConnectorError
+import json
+import requests
+
+logger = get_logger('ip-api')
+
+
+class IPAPI(object):
+
+    def __init__(self, config):
+        self.url = config.get('server').strip()
+        if not self.url.startswith('http://'):
+            self.url = 'http://' + self.url
+        if self.url[-1] == '/':
+            self.url = self.url[:-1]
+        self.headers = {'accept': 'application/json'}
+
+    def make_rest_call(self, endpoint, method='POST', data=None, headers=None):
+        url = self.url + endpoint
+        logger.info('Final url to make rest call is: {0}'.format(url))
+        if headers:
+            self.headers.update(headers)
+        if data:
+            logger.info('Converting the data: {0} into an equivalent JSON object.'.format(data))
```

Comment:
```suggestion
            logger.debug('Converting the data: {0} into an equivalent JSON object.'.format(data))
```
use debug flag. this will avoid unnecessary logging.